public/app/percona/add-instance/components/AddInstance/AddInstance.test.tsx

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
-import { ReactWrapper, shallow, ShallowWrapper } from 'enzyme';
-import { AddInstance, SelectInstance } from './AddInstance';
+import { AddInstance } from './AddInstance';
 import { instanceList } from './AddInstance.constants';
-import { getMount } from 'app/percona/shared/helpers/testUtils';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
 
 jest.mock('app/percona/settings/Settings.service');
 
 describe('AddInstance page::', () => {
-  it('should render a given number of links', () => {
-    const wrapper: ShallowWrapper = shallow(<AddInstance onSelectInstanceType={() => {}} />);
+  it('should render a given number of links', async () => {
+    await waitFor(() => render(<AddInstance onSelectInstanceType={() => {}} />));
```

Comment:
Do we really need `waitFor` here?

Try `findAllByRole` instead of `getAllByRole`.

Also one tip: it's always best try and use testid as much as possible instead of element types (I'm victim of this too :D)

Reply:
yes, you're right, I just realized that it works, usually, I try to render something before running tests, without tests. And in my case, it didn't work. But when I put everything together with find, it got better! I will refactor it